statsmodels/base/tests/test_generic_methods.py

```diff
@@ -284,8 +284,12 @@ def test_zero_collinear(self):
             # cov_type is otherwise ignored
             if cov_type != 'nonrobust' and (isinstance(self.results.model,
                                                        sm.RLM)):
-                return
+                continue
 
+            # method_kwd is not used for GLM, mle_settings is now added
+            # is not used currently, could be used for OLS, WLS
```

Comment:
not/now/not wording has weird flow.  can the second line get its own subject to be independently sentence-like?